### PR TITLE
[Isthmus]enable tpch queries tpch_smoke.sh

### PR DESCRIPTION
**Pull request by jikunshang**

### CODE REVIEW

**Comment by jikunshang** [comment]
`1 workflow awaiting approval` @jacques-n  can you approve it to enable CI workflow? thanks!

File: isthmus/src/test/resources/tpch/queries/15.sql [hunk 1]
```diff
@@ -1,4 +1,3 @@
--- converted to CTE since DDL is not part of Substrait.
```
Comment (JamesRTaylor) [inline isthmus/src/test/resources/tpch/queries/15.sql:1]: My preference would be to figure out why comments are causing a problem

Comment (jikunshang) [reply isthmus/src/test/resources/tpch/queries/15.sql:1]: My guess is Isthmus will treat the dash here as input parameter and it throws `Missing required parameter: '<sql>'` message.

**Comment by JamesRTaylor** [comment]
We can commit this patch is you remove your changes around getting rid of the comments (i.e. just keep the change that registers the missing proto function for reflection), and adjust the passing tests.

**Comment by jikunshang** [comment]
> We can commit this patch is you remove your changes around getting rid of the comments (i.e. just keep the change that registers the missing proto function for reflection), and adjust the passing tests.

updated.

File: isthmus/src/test/resources/tpch/queries/15.sql (hunk 1 above)
Comment (JamesRTaylor) [reply isthmus/src/test/resources/tpch/queries/15.sql]: Please file an issue for the comment in the sql file breaking when using native image

**Comment by JamesRTaylor** [comment]
Thanks for the contribution, @jikunshang

**Pull request merged**